tests/test_scenarios.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env python
+import logging
+from pathlib import Path
+
+import pytest
+import requests
+from conftest import TEST_PATH
+from pyrdfj2 import J2RDFSyntaxBuilder
+from pyrdfstore import RDFStore, create_rdf_store
+from rdflib import Graph
+from util4tests import run_single_test
+
+from travharv import TravHarv
+from travharv.store import RDFStoreAccess
+
+log = logging.getLogger(__name__)
+
+QUERY_BUILDER: J2RDFSyntaxBuilder = J2RDFSyntaxBuilder(
+    templates_folder=str(Path(__file__).parent / "scenarios" / "templates")
+)
+
+BASE = "http://localhost:8080/"
+OUTPUTS = TEST_PATH / "output"
+INPUT = TEST_PATH / "input"
+CONFIGS = TEST_PATH / "travharv_config"
+RDF_MIMES = {
+    "text/turtle",
+    "application/ld+json",
+}
+
+# TODO write out all the assertions for the tests
+# this being the number of triples
+# the documents that ware asserted
+SCENARIOS_OUTCOMES = {
+    "dereference_test1_sparql.yml": [
+        {"expected_len_triples": "19", "mode": "stop"},
+        {"expected_len_triples": "19", "mode": "nostop"},
+    ],
+    "dereference_test2_sparql.yml": [
+        {"expected_len_triples": "28", "mode": "stop"},
+        {"expected_len_triples": "TBD", "mode": "nostop"},
+    ],
+    "dereference_test3_sparql.yml": [
+        {"expected_len_triples": "19", "mode": "stop"},
+        {"expected_len_triples": "TBD", "mode": "nostop"},
+    ],
+    "dereference_test4_sparql.yml": [
+        {"expected_len_triples": "19", "mode": "stop"},
+        {"expected_len_triples": "TBD", "mode": "nostop"},
+    ],
+    "dereference_test5_sparql.yml": [
+        {"expected_len_triples": "19", "mode": "stop"},
+        {"expected_len_triples": "TBD", "mode": "nostop"},
+    ],
+}
+
+
+# test if all objects can be retrieved
+@pytest.mark.usefixtures("httpd_server_base", "all_extensions_testset")
+def test_inputs(httpd_server_base: str, all_extensions_testset):
```

Comment:
more a test of the httpd and he conftest.py then an actual 'scenario'

would remove or place in a separate test_conf_fixtures.py